{{cookiecutter.project_name}}/CONTRIBUTING.md

```diff
@@ -1,26 +1,24 @@
-Contributor Guide
-=================
+# Contributor Guide
 
 Thank you for your interest in improving this project.
-This project is open-source under the `{{cookiecutter.license.replace("-", " ")}} license`_ and
+This project is open-source under the [{{cookiecutter.license.replace("-", " ")}} license][] and
 welcomes contributions in the form of bug reports, feature requests, and pull requests.
 
 Here is a list of important resources for contributors:
 
-- `Source Code`_
-- `Documentation`_
-- `Issue Tracker`_
-- `Code of Conduct`_
+- [Source Code][]
```

Comment:
Can all the `[]` be removed here? I believe I saw this in previous commits.

Reply:
Same as mentioned above 

I have changed this kind of error before. 

[this](https://github.com/cauliyang/cookiecutter-hypermodern-python/edit/main/%7B%7Bcookiecutter.project_name%7D%7D/CONTRIBUTING.md)